distribution/scripts/jmeter/run-performance-tests.sh

```diff
@@ -70,7 +70,7 @@ declare -A test_scenario2=(
     [skip]=false
 )
 declare -A test_scenario3=(
-    [name]="transformation_https"
+    [name]="https_transformation"
```

Comment:
Should `https_transformation` be renamed as `h1_transformation`? Similarly `http_transformation` as `h1c_transformation`?

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-performance/pull/23/commits/7348d0132de911748e34cb165d0c0ac5a539f399